ksqldb-engine/src/main/java/io/confluent/ksql/engine/QueryCleanupService.java

```diff
@@ -120,7 +121,7 @@ public String getQueryId() {
 
     @Override
     public void run() {
-      if (queryId.equals("")) {
+      if (!queryId.isPresent()) {
```

Comment:
can we at least leave a comment here explaining what it means for the queryId to be empty?

Reply:
I think I am going to change the name from `queryId` to `topologyName` one its more accurate and two it seems to be pretty confusing.